src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/YesNoTest.java

```diff
@@ -1,74 +1,77 @@
 package uk.gov.hmcts.reform.sscs.ccd.domain;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.NO;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.YES;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isNoOrNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-public class YesNoTest {
+class YesNoTest {
```

Comment:
Is there any reason you made this class package private ?

Reply:
JUnit5 is more tolerant regarding the visibilities of Test classes than JUnit4, which required everything to be public.
In this context, JUnit5 test classes can have any visibility but private, however, it is recommended to use the default package visibility, which improves readability of code.

(Description from SonarLint)